### ISSUE TITLE
Pretrained Weights and Scope

### ISSUE THREAD

**Issue by ajaybati** [Z-Zheng/pytorch-change-models#10]
Will pretrained weights for these models be released somewhere? I was looking for some of the trained models such as ChangeStar (original) or ChangeStar (with ChangenV2 data).

The other question I had was a lot of these models are related to satellite imagery or general earth science-related tasks. I was looking to run inference for some of these models on the natural image domain. I see for example ChangeNetV2, Segment Any Change, and some others do pretty well on the natural image domain. However, to test this, I would need the pretrained weights for the above mentioned models (changestar, change mask, etc.).

**Comment by Z-Zheng** [comment]
Changen2 pre-trained ChangeStar models can be found here https://github.com/Z-Zheng/pytorch-change-models/blob/9c564feecfb577069b9a9bc5859264f768581046/torchange/models/changen2/_changestar_1x256.py#L60

Using these APIs, it will automatically download the weight from Huggingface.

Segment Any Change just used SAM's weight. So, directly downloading their weights will work well https://github.com/facebookresearch/segment-anything

**Comment by ajaybati** [comment]
1. Thanks so much for the prompt reply! Got it, I'll check it out. But, are there other weights for the rest of the models? For example, I saw there is ChangeMask, ChangeStar2, and more listed in your model zoo. Are there available checkpoints for these models? And yes, I saw that SAM is just no training heuristic based.

2. Just wanted to ask based on your experience how do these models do on natural image domains?

**Comment by Z-Zheng** [comment]
No, we do not plan to release other dataset-tailored weights because re-training them is easy. For 2, it depends on your data. ChangeMask and ChangeStar architectures generally work well on our projects. Recommend you to use.

**Comment by ajaybati** [comment]
Are there any pretrained weights that are open sourced for either ChangeStar or ChangeMask that you think would work well for natural image domains?
